Criollo/Source/HTTP/CRHTTPConnection.m

```diff
@@ -136,6 +136,10 @@ - (void)socket:(GCDAsyncSocket *)sock didReadData:(NSData*)data withTag:(long)ta
                     CRHTTPVersion version = CRHTTPVersionMake(versionSpec);
 
                     NSRange rangeOfHostHeader = [data rangeOfData:[@"Host: " dataUsingEncoding:NSUTF8StringEncoding] options:0 range:NSMakeRange(0, data.length)];
+					
+                    // Since HTTP header fields are case-insensitive (https://www.w3.org/Protocols/rfc2616/rfc2616-sec4.html#sec4.2), we have to check for both `Host` and `host`.
```

Comment:
This will solve that case, but what if somebody decides to go with more creative approaches. `HosT`, aso. We should compare it lowercased. But that is for the future. This should solve immediate issues

Reply:
What about converting `data` to a string, setting it all to lowercase, and then checking for `host: ` against that? Do you think that would be a better long-term solution?